### PR TITLE
Connect hdfs test with Hive

### PR DESCRIPTION
**Pull request by Ishiihara**

### CODE REVIEW

**Comment by Ishiihara** [comment]
@ewencp Added the connect test with hive.

File: muckrake/tests/connect_hdfs_test.py [hunk 1]
```diff
@@ -298,3 +303,86 @@ def verify_no_compatibility(self):
         for filename, schema in self.partition_avro_schemas(self.topic, self.partition):
             assert index % 2 + 1 == int(schema['connect.version'])
             index += 1
+
+
+class ConnectHiveTest(ConnectHdfsTest):
```
Comment (ewencp) [inline muckrake/tests/connect_hdfs_test.py]: Did you intend to inherit from `ConnectHdfsTest` or should this be `ConnectHdfsTestBase`? As is, it's inheriting all the tests methods from `ConnectHdfsTest`.

**Comment by Ishiihara** [comment]
@ewencp Updated the PR to inherit from ConnectHdfsTestBase.

**Comment by Ishiihara** [comment]
@ewencp Is this good to merge?

**Comment by ewencp** [comment]
Sorry, had to fix the Hive issue and then wanted to run the test. LGTM.

**Pull request merged**